app/Controllers/Http/BoardPlayersController.ts

```diff
@@ -13,17 +14,36 @@ export default class BoardPlayersController {
     if (board.players.length === board.maxPlayers) {
       throw new BadRequestException('All seats have already been filled!')
     }
-
+    const user = await User.find(params.player_id)
+    if (!user) {
+      return response.notFound('User not found!')
+    }
+    await bouncer.with('BoardPlayerPolicy').authorize('create', board, user)
```

Comment:
We don't need to send the creation data again, send only the pending user id, check if it exists, and if exists store in players